include/Command.h

```diff
@@ -0,0 +1,42 @@
+#ifndef BASH_COMMAND_H
+#define BASH_COMMAND_H
+
+// Containers
+#include <string>
+#include <vector>
+#include <experimental/filesystem>
```

Comment:
If you upgrade to C++17 and above then you can use the standard `filesystem` lib